gdp/column/column.py

```diff
@@ -50,261 +75,394 @@ def build_column(min_trays, max_trays, xD, xB, x_input, nlp_solver, provide_init
                     'D': 0, 'E': 0}}
-    m.dH_vap = {'benzene': 33.770E3, 'toluene': 38.262E3}  # J/mol
+    # Heat of vaporization for each component [J/mol]
+    m.dH_vap = {'benzene': 33.770E3, 'toluene': 38.262E3}  
 
+    # Define set of potential trays
     m.trays = RangeSet(max_trays, doc='Set of potential trays')
+    # Define set of trays that can be turned on and off
     m.conditional_trays = Set(
         initialize=m.trays - [m.condens_tray, m.feed_tray, m.reboil_tray],
         doc="Trays that may be turned on and off.")
+    # Disjunct for tray existence
     m.tray = Disjunct(m.conditional_trays, doc='Disjunct for tray existence')
+    # Disjunct for tray absence
     m.no_tray = Disjunct(m.conditional_trays, doc='Disjunct for tray absence')
 
-    @m.Disjunction(m.conditional_trays, doc='Tray exists or does not')
+    # Disjunction statement defining whether a tray exists or not
+    @m.Disjunction(m.conditional_trays, doc='Tray exists or does not') 
     def tray_no_tray(b, t):
+        """Disjunction statement defining whether a tray exists or not"""""
         return [b.tray[t], b.no_tray[t]]
+
+    # Constraint for minimum number of trays, adding 1 for feed tray
     m.minimum_num_trays = Constraint(
         expr=sum(m.tray[t].indicator_var
                  for t in m.conditional_trays) + 1  # for feed tray
         >= min_trays)
 
+    # If user provides initialization values, use them
     if provide_init:
+        # Feed temperature variable [K]
         m.T_feed = Var(
             doc='Feed temperature [K]', domain=NonNegativeReals,
             bounds=(min_T, max_T), initialize=init['T_feed'])
+
+        # Vapor fraction of feed variable
         m.feed_vap_frac = Var(
             doc='Vapor fraction of feed',
             initialize=init['feed_vap_frac'], bounds=(0, 1))
+    
+        # Component feed flow variable (mol/s)
         m.feed = Var(
             m.comps, doc='Total component feed flow [mol/s]', initialize=init['feed'])
+    
+        # Liquid mole fraction variable
         m.x = Var(m.comps, m.trays, doc='Liquid mole fraction',
                   bounds=(0, 1), domain=NonNegativeReals, initialize=init['x'])
+    
+        # Vapor mole fraction variable
         m.y = Var(m.comps, m.trays, doc='Vapor mole fraction',
                   bounds=(0, 1), domain=NonNegativeReals, initialize=init['y'])
+    
+        # Component liquid flows from tray variable [kmol]
         m.L = Var(m.comps, m.trays,
-                  doc='component liquid flows from tray in kmol',
+                  doc='component liquid flows from tray [kmol]',
                   domain=NonNegativeReals, bounds=(0, max_flow),
                   initialize=init['L'])
+
+        # Component vapor flows from tray variable [kmol]
         m.V = Var(m.comps, m.trays,
-                  doc='component vapor flows from tray in kmol',
+                  doc='component vapor flows from tray [kmol]',
```

Comment:
Same as above, review all flows and their units

Reply:
I changed from [kmol] to [kmol/s]. But I am not sure of changing the units.